recipes/s3fs/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.0.4" %}
+
+package:
+  name: s3fs
+  version: "{{ version }}"
+
+source:
+  fn: s3fs-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/s3fs/s3fs-{{ version }}.tar.gz
+  md5: 4fa1bcec029283a904218544c1fa5eab
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
```

Comment:
When using `--single-version-externally-managed --record=record.txt` we need to specify `setuptools` at build time. (I am actually puzzled that this worked without it.)